bootstrap/css/index.css

```diff
@@ -0,0 +1,320 @@
+*{
+  font-family: "Roboto Condensed", sans-serif;
+  box-sizing: border-box;
+}
+
+.top-story-fig h2{
+  font-size: 1.375rem;
+  font-weight: 700;
+}
+
+.top-story-fig h3{
+  font-size: 1.25rem;
+  font-weight: 700;
+}
+
+.top-story-fig h2 a{
+  color:#000;
+}
+
+header{
+  position: fixed;
+  width: 100%;
+  z-index: 2;
+}
+
+.nav-large-xlarge li a{
+  cursor:pointer;
+  color: black;
+}
+
+.nav-large-xlarge li a:hover{
+  color: red;
+  text-decoration: none;
+}
+
+{
+  cursor: pointer;
```

Comment:
There is not a reference for this property value definition, throwing an error in linter. Please fix 😅